modules/microshift-nw-ipv6-dual-stack-migrating-config.adoc

```diff
@@ -0,0 +1,149 @@
+
+You can convert a single-stack cluster to dual-stack cluster networking that supports IPv4 and IPv6 address families by setting two entries in the service and cluster network parameters in the {microshift-short} configuration file.
+
+* The first IP family in the configuration is the primary IP stack in the cluster.
+* {microshift-short} system pods and services are automatically updated upon {microshift-short} restart.
+* After the cluster is migrated to dual-stack networking and has restarted, enable workload pods and services for dual-stack networking by restarting them.
+
+[IMPORTANT]
+====
+The OVN-Kubernetes network plugin requires that both IPv4 and IPv6 default routes be on the same network device. IPv4 and IPv6 default routes on separate network devices is not supported.
+====
+
+[IMPORTANT]
+====
+When using dual-stack networking where IPv6 is required, you cannot use IPv4-mapped IPv6 addresses, such as `::FFFF:198.51.100.1`.
+====
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You have root access to the cluster.
+* Your cluster uses the OVN-Kubernetes network plugin.
+* The host has both IPv4 and IPv6 addresses and routes, including a default for each.
+* The host has at least two L3 networks, IPv4 and IPv6.
+
+.Procedure
+
+. If you have not done so, make a copy of the provided `config.yaml.default` file in the `/etc/microshift/` directory, renaming it `config.yaml`.
+
+. Keep the new {microshift-short} `config.yaml` in the `/etc/microshift/` directory. Your `config.yaml` file is read every time the {microshift-short} service starts.
++
+[NOTE]
+====
+After you create it, the `config.yaml` file takes precedence over built-in settings.
+====
+
+. Add IPv6 configurations to the `network` section of the {microshift-short} YAML with your desired valid values:
++
+[WARNING]
+====
+You must keep the same first entry across restarts and migrations. This is true for any migration, single-to-dual stack, or dual-to-single stack. A complete wipe of the etcd database is required if a change to the first entry is needed. This might result in application data loss and is not supported.
+====
++
+.. Add an IPv6 configuration for a second network in the `network` section of the {microshift-short} YAML with your desired valid values.
+
+.. Add network assignments to the `network` section of the {microshift-short} `config.yaml` to enable dual stack with IPv6 as secondary network.
++
+.Example dual-stack `IPv6` configuration with network assignments
++
+[source,terminal]
+----
+# ...
+apiServer:
+  subjectAltNames:
+  - 192.168.113.117
+  - 2001:db9:ca7:ff::1db8 <1>
+network:
+  clusterNetwork:
+  - 10.42.0.0/16 <2>
+  - fd01::/48 <3>
+  serviceNetwork:
+  - 10.43.0.0/16
+  - fd02::/112 <4>
+node:
+  nodeIP: 192.168.113.117 <5>
+  nodeIPv6: 2001:db9:ca7:ff::1db8 <6>
+# ...
+----
+<1> The IPv6 node address.
+<2> Specify a `clusterNetwork` with a CIDR value that is less than `64`.
```

Comment:
Missed this one, sorry. Max value for ipv4 mask must is 24. In ipv6 case its 64.